docs/man/docker-ps.1.md

```diff
@@ -1,6 +1,6 @@
 % DOCKER(1) Docker User Manuals
 % Docker Community
-% JUNE 2014
+% FEBRURAY 2015
```

Comment:
Sorry, that's short for "you've got a typo; 'februray' should be 'february'"

(Writing it as `s/something/something else/` is a shorter way to write that, it's inspired by the [`sed utility`](http://www.grymoire.com/Unix/Sed.html#uh-2))